content/learning-paths/smartphones-and-mobile/ray_tracing/rt09_optimizing.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Optimizing"
+weight: 10
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Optimizing
+
+We recommend developers to check [our ray tracing best practices](https://developer.arm.com/documentation/101897/0302/Ray-tracing) to learn how to optimize ray tracing content. We also greatly recommend that you profile early and often, since ray tracing can be very costly.
```

Comment:
best practice links: should be https://developer.arm.com/documentation/101897/latest/Ray-tracing and similar, with "latest" rather than a version number "0302", so that they update with new best practice versions